lib/spack/spack/cmd/install.py

```diff
@@ -122,6 +123,11 @@ def setup_parser(subparser):
         default=None,
         help="filename for the log file. if not passed a default will be used"
     )
+    subparser.add_argument(
+        '--log-upload-site',
```

Comment:
Is this just for a CDash site?  Might be good to make it `--cdash-upload-url` or something more specific.